### PR TITLE
Fix lakes and rivers geometry on scoped geo subplots

### PR DESCRIPTION
**Pull request by etpinard**
fixes https://github.com/plotly/plotly.js/issues/4046 (a regression from https://github.com/plotly/plotly.js/pull/3856) with the help of https://github.com/etpinard/sane-topojson/pull/14

### CODE REVIEW

**Comment by etpinard** [comment]
cc @plotly/plotly_js don't attempt to update https://codepen.io/nicolaskruchten/pen/orRBKb?editors=0010 from https://github.com/plotly/plotly.js/issues/4046 to see the new behaviour. We need to first update https://cdn.plot.ly/africa_110m.json and friends.

**Comment by etpinard** [comment]
You'll notice that `scope: 'europe'` is also currently having issues with `showlakes:true`.

File: test/image/mocks/geo_lakes-and-rivers.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
```
Comment (etpinard) [inline test/image/mocks/geo_lakes-and-rivers.json]: This mock currently looks like:

newplot (42)

Comment (etpinard) [reply test/image/mocks/geo_lakes-and-rivers.json]: Using topojson files from before https://github.com/plotly/plotly.js/pull/3856 fixes the lakes polygon, but the scoped rivers were still off, see https://github.com/plotly/plotly.js/pull/4048/commits/7075d136b08890741f454873d99b350e4fed7045 or diff in https://github.com/plotly/plotly.js/pull/4048/commits/5385673ed988d82dbab4be399bfce6b1f63bc01a

**Comment by archmoj** [comment]
It is rather difficult to view & compare the before/after images (with aspect-ratio close to 2) on small monitors. @etpinard would you mind splitting the mock into two mocks?

**Comment by etpinard** [comment]
@archmoj does this help?

Peek 2019-07-15 15-55

**Comment by archmoj** [comment]
@etpinard I cannot `swipe` over the center (mid-height) of the image. 
Anyway that's just fine.

**Comment by antoinerg** [comment]
Thank you @etpinard for the update and the mock

:dancer:

**Comment by etpinard** [comment]
Is @archmoj ok also with the changes?

Here's the script I used to generate the new mock:

```js
var schema = Plotly.PlotSchema.get()
var scopes = schema.layout.layoutAttributes.geo.scope.values
var resolutions = [110, 50]

var traces = []
var layout = {
  grid: {rows: scopes.length, columns: resolutions.length},
  width: 600,
  height: 1500
}
var k = 0

resolutions.forEach((r, i) => {
scopes.forEach((s, j) => {
  var id = 'geo' + (k ? (k + 1) : '')
  
  traces.push({type: 'scattergeo', geo: id})
  layout[id] = {
    scope: s,
    resolution: r,
    domain: {column: i, row: j},
    landcolor: "brown",
    showland: true,
    lakecolor: "blue",
    showlakes: true,
    rivercolor: "blue",
    showrivers: true
  }
  
  k++
})
})

Plotly.newPlot(gd, traces, layout, {scrollZoom:false})
```

It should be easy to transpose (i,j) -> (j,i), if that makes it easier to review for you.

**Comment by archmoj** [comment]
:dancer:

**Pull request merged**